### PR TITLE
App can now properly distinguish between online users

### PR DESCRIPTION
**Pull request by Winterlord8**
Hijacked FirebaseAuth onAuthStateChanged function to track when a user is online or not, changes are made to the onlineUsers database.

### CODE REVIEW

**Review by vrivero09 — APPROVED** [review]
Looks good! Should try to remove the unnecessary usernames so we can get an accurate list.

File: backend/firebase.js [hunk 1]
```diff
@@ -235,10 +247,8 @@ class Firebase {
             })
 }
 
-getImageRemoteUri = (image, cme) => {
-
-  const photoPath = `uploads/${firebase.auth().currentUser.uid}/${cme.id}.jpg`;
-
+getImageRemoteUri = (image) => {
+  const photoPath = `uploads/${Date.now()}.jpg`;
```
Comment (vrive) [inline backend/firebase.js:251]: change this back please!

const photoPath = `uploads/${firebase.auth().currentUser.uid}/${cme.id}.jpg`;

**Review by vrive — APPROVED** [review]
Works great!
Just make sure to remove old code that you dont use and also undo the change you made to the 'getImageRemoteUri' function

**Comment by vrive** [comment]
can you run 'git pull origin master' and then 'npm install' and then push? I think that might help with the conflict

**Review by vrivero09 — CHANGES_REQUESTED** [review]
Reviewing the request and when I go to profile>CME screen, it gives me an error about the certs. Just need to delete all certs definition in profile and login screen and leave the navigation to cme in profile by itself:

line 89

  const handleCerts = () => {
    props.navigation.navigate('CME')
  }

**Review by vrivero09 — APPROVED** [review]

**Pull request merged**